lib/formulas/resource-router-mapping.js

```diff
@@ -30,9 +30,11 @@ function traverseExpressions(routes) {
     if (isResourceRoute(route)) {
       transformResourceToRoute(route);
     }
-    var lastArgument = route.expression.arguments[route.expression.arguments.length-1];
-    if (types.FunctionExpression.check(lastArgument)) {
-      traverseExpressions(lastArgument.body.body);
+    if (route.expression) {
```

Comment:
Can you also check if the AST type is a `CallExpression` with `if (types.CallExpression.check(route.expression)) {` ?